package.json

```diff
@@ -1,18 +1,33 @@
-{ "name" : "jsonpointer"
-, "description" : "Simple JSON Addressing."
-, "tags" : ["util", "simple", "util", "utility"]
-, "version" : "1.0.1"
-, "author" : "Jan Lehnardt <jan@apache.org>"
-, "contributors":
-  ["Joe Hildebrand <joe-github@cursive.net>"]
-, "repository" :
-  { "type" : "git"
-  , "url" : "http://github.com/janl/node-jsonpointer.git"
+{
+  "name": "jsonpointer",
+  "description": "Simple JSON Addressing.",
+  "tags": [
+    "util",
+    "simple",
+    "util",
+    "utility"
+  ],
+  "version": "1.0.1",
+  "author": "Jan Lehnardt <jan@apache.org>",
+  "contributors": [
+    "Joe Hildebrand <joe-github@cursive.net>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/janl/node-jsonpointer.git"
+  },
+  "bugs": {
+    "url": "http://github.com/janl/node-jsonpointer/issues"
+  },
+  "engines": [
+    "node >= 0.4.9"
+  ],
+  "main": "./jsonpointer",
+  "scripts": {
+    "test": "node test.js"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "console": "^0.5.2"
```

Comment:
This is only needed when running the tests in browsers without console.log. So maybe we can remove it completely and just use `console.log` directly.